travis/hibench.conf

```diff
@@ -59,5 +76,63 @@ hibench.nutch.home		${hibench.home}/hadoopbench/nutchindexing/target/${hibench.n
+hibench.streambench.sampleProbability 0.1
+hibench.streambench.fixWindowDuration            10000
+hibench.streambench.fixWindowSlideStep           10000
+
+
+#======================================================
+# Kafka for streaming benchmarks
+#======================================================
+hibench.streambench.kafka.home                  /PATH/TO/YOUR/KAFKA/HOME
+# zookeeper host:port of kafka cluster, host1:port1,host2:port2...
+hibench.streambench.zkHost
+# Kafka broker lists, written in mode host:port,host:port,..
+hibench.streambench.kafka.brokerList
+hibench.streambench.kafka.consumerGroup          HiBench
+# number of partitions of generated topic (default 20)
+hibench.streambench.kafka.topicPartitions       20
+# consumer group of the consumer for kafka (default: HiBench)
+hibench.streambench.kafka.consumerGroup HiBench
+# Set the starting offset of kafkaConsumer (default: largest)
+hibench.streambench.kafka.offsetReset largest
+
+
+#======================================================
+# Data generator for streaming benchmarks
+#======================================================
+# Interval span in millisecond (default: 50)
+hibench.streambench.datagen.intervalSpan         50
+# Number of records to generate per interval span (default: 5)
+hibench.streambench.datagen.recordsPerInterval   5
+# fixed length of record (default: 200)
+hibench.streambench.datagen.recordLength         200
+# Number of KafkaProducer running on different thread (default: 1)
+hibench.streambench.datagen.producerNumber       1
+# Total round count of data send (default: -1 means infinity)
+hibench.streambench.datagen.totalRounds          -1
+# Number of total records that will be generated (default: -1 means infinity)
+hibench.streambench.datagen.totalRecords        -1
+# default path to store seed files (default: ${hibench.hdfs.data.dir}/Streaming)
+hibench.streambench.datagen.dir                         ${hibench.hdfs.data.dir}/Streaming
+# default path setting for genearate data1 & data2
+hibench.streambench.datagen.data1.name                  Seed
+hibench.streambench.datagen.data1.dir                   ${hibench.streambench.datagen.dir}/${hibench.streambench.datagen.data1.name}
+hibench.streambench.datagen.data2_cluster.dir           ${hibench.streambench.datagen.dir}/Kmeans/Cluster
+hibench.streambench.datagen.data2_samples.dir           ${hibench.streambench.datagen.dir}/Kmeans/Samples
+
+#======================================================
+# MetricsReader for streaming benchmarks
+#======================================================
+# Number of sample records for `MetricsReader` (default: 5000000)
+hibench.streambench.metricsReader.sampleNum      5000000
+# Number of thread for `MetricsReader` (default: 20)
+hibench.streambench.metricsReader.threadNum      20
+# The dir where stored the report of benchmarks (default: ${hibench.home}/report)
+hibench.streambench.metricsReader.outputDir      ${hibench.home}/report
```

Comment:
Let's remove the streambench part due to we don't run streaming benchmark in CI.

Reply:
@gczsjdy  it's done.